config/homebucket/clusterworkspacetype-use-home-clusterrolebinding.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
in general, we have a naming convention for these files. The resource comes first, not last.